Jenkinsfile

```diff
@@ -66,24 +74,6 @@ pipeline {
                 '''
             }
         }
-
-        stage('Lint Integration Tests') {
-            steps {
-                sh './bin/lint_tests'
-            }
-        }
-
-        stage('Lint Spec File') {
-            steps {
-                sh './bin/lint_spec'
-            }
-        }
-        
-        stage('API Contract Test') {
-            steps {
-                sh './bin/api_test'
-            }
-        }
```

Comment:
why are we removing these? there is no note in the PR or commit messages as far as I can tell that explains this, but we should give a clear reason for removing tests

Reply:
I actually removed them based on a comment you had made a while ago about duplicating test runs on Jenkins and GH actions. The reason I removed them is because we already run them in actions and there is no benefit to running them again in Jenkins so it is just using up more CI resources then it needs to